frontends/verilog/verilog_frontend.cc

```diff
@@ -492,6 +492,11 @@ struct VerilogFrontend : public Frontend {
 		frontend_verilog_yyparse();
 		frontend_verilog_yylex_destroy();
 
+		for (auto it : user_type_stack) {
+		        delete it;
+		}
+		user_type_stack.clear();
```

Comment:
I'm not sure about this one, should these types be kept between different frontend invocations?

Reply:
CI answered that for me, I moved it to the pass destructor.